tox.ini

```diff
@@ -57,6 +57,7 @@ deps =
     jsonschema
     psycopg2-binary
     pytest
+    pytest-asyncio
```

Comment:
We don't really need this for the unit tests, but otherwise pytest will complain about the dangling config option. Removing the option breaks the integration tests.

Reply:
Good to commit it to all repos, tests prodices warning about it everywhere. Resolved?